haret/src/vr/states/common/normal.rs

```diff
@@ -67,12 +67,11 @@ pub fn handle_start_view<T: State>(state: T,
 }
 
 pub fn handle_get_state<T: State>(state: T,
-                                  msg: GetState,
+                                  GetState { epoch, view, op }: GetState,
```

Comment:
Lol, I wouldn't go that far @mrallen1 . Pattern matching has existed via `match` statements forever, but I didn't realize it was also possible in fn clauses.